docs/Deploy/enable-subscription-creation.md

```diff
@@ -28,9 +28,7 @@ $spnObjectId = ""
 $currentContext = Get-AzContext
```

Comment:
"$currentContext" can also be removed

Reply:
Pls see comment from Hansjoerg

Reply:
Yes the variable currentContext is obsolete now.

Reply:
Awesome thank you @stefanstranger